api-report/container-definitions.api.md

```diff
@@ -479,10 +481,12 @@ export interface IResolvedFluidCodeDetails extends IFluidCodeDetails {
 // @public
 export interface IRuntime extends IDisposable {
     createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree;
+    readonly entryPoint?: Promise<FluidObject | undefined>;
     getPendingLocalState(): unknown;
     notifyAttaching(snapshot: ISnapshotTreeWithBlobContents): void;
     process(message: ISequencedDocumentMessage, local: boolean): any;
     processSignal(message: any, local: boolean): any;
+    // @deprecated
```

Comment:
feels a bit adhoc to deprecate this here. maybe another pr that just deprecates request everywhere is a better path once this is all in

Reply:
for instance, its not deprecated on the loader

Reply:
Ah, missed the fact that the Loader also has request(). But I'm ok moving that to a separate PR, I'll revert this.